YesterdayApi/Startup.cs

```diff
@@ -25,9 +26,19 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
+
+            services.AddAuthentication("Bearer")
+                .AddIdentityServerAuthentication(options =>
+                {
+                    options.Authority = "http://localhost:5000";
```

Comment:
Maybe would be better to put this url into the configuration?

Reply:
Yeah.

Reply:
Yeah.